homeassistant/components/lunatone/light.py

```diff
@@ -82,6 +90,28 @@ def is_on(self) -> bool:
         """Return True if light is on."""
         return self._device is not None and self._device.is_on
 
+    @property
+    def brightness(self) -> int:
+        """Return the brightness of this light between 0..255."""
+        brightness = 0
+        if self._device is not None:
+            brightness = value_to_brightness(
+                self.BRIGHTNESS_SCALE, self._device.brightness
+            )
+        return brightness
+
+    @property
+    def color_mode(self) -> ColorMode:
+        """Return the color mode of the light."""
+        if self._device is not None and self._device.is_dimmable:
+            return ColorMode.BRIGHTNESS
+        return ColorMode.ONOFF
+
+    @property
+    def supported_color_modes(self) -> set[ColorMode]:
+        """Return the supported color modes."""
+        return {self.color_mode}
```

Comment:
Shouldn't this be the supported color modes, rather than the current one?
```suggestion
        return self.supported_color_modes
```

Reply:
Ok, agreed.